src/chart/barChartMultiSeries.js

```diff
@@ -14,7 +14,7 @@ import component from "../component";
  * var myChart = d3.x3dom.chart.barChartMultiSeries();
  * chartHolder.datum(myData).call(myChart);
  */
-export default function() {
+export default function barChartMultiSeries() {
```

Comment:
Yes, this makes sense. The only downside is that when using an anonymous function, you lose the ability to view the function name in stack traces, when debugging etc. But this may just be a minor inconvenience.